redaxo/src/core/lib/fragment.php

```diff
@@ -130,7 +129,7 @@ public function parse($filename)
             }
         }
 
-        throw new rex_exception(sprintf('Fragmentfile "%s" not found!', $filename));
+        throw new InvalidArgumentException(sprintf('Fragmentfile "%s" not found!', $filename));
```

Comment:
im anderen fehlerfall in der gleichen methode verwenden wir auch `InvalidArgumentException`.
und der case hier ist ebenfalls ein programmierer fehler, daher in meinen augen auch hier passend